recipes/iiboost/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "iiboost" %}
+{% set version = "0.2" %}
+{% set sha256 = "0ecd906b453a6a278651e324352bd6f820b70aa4ca15fa42843bdb5c01ee7079" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/cbecker/iiboost/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+  patches:
+    - cmath_fixes.diff
+    - python3.diff
+
+build:
+  number: 0
+  skip: True        # [win]
+  features:
+    - vc14          # [win and py35]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - libgcc                # [linux]
```

Comment:
I replaced ``libgcc`` with ``gcc`` in the build requirements and left ``libgcc`` in the run requirements. Is this ok?